packages/web-components/src/components/masthead/masthead-profile.ts

```diff
@@ -108,9 +108,6 @@ class DDSMastheadProfile extends HostListenerMixin(FocusMixin(StableSelectorMixi
   }
 
   connectedCallback() {
```

Comment:
I think we can go ahead and remove the entire `connectedCallback()` function

Reply:
Thanks, I made a new commit without the function.